com.geekbench.Geekbench6.yml

```diff
@@ -0,0 +1,33 @@
+app-id: com.geekbench.Geekbench6
+runtime: org.freedesktop.Platform
+runtime-version: "22.08"
+sdk: org.freedesktop.Sdk
+command: geekbench6
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: geekbench
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/bin
+      - cp -r geekbench/* ${FLATPAK_DEST}/bin/
+      - install -Dm755 geekbench/geekbench6 ${FLATPAK_DEST}/geekbench6
+      - install -Dm644 app-icon.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg
+      - install -Dm644 com.geekbench.Geekbench6.appdata.xml ${FLATPAK_DEST}/share/appdata/${FLATPAK_ID}.appdata.xml
+      - install -Dm644 com.geekbench.Geekbench6.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+    sources:
+      - type: file
+        path: app-icon.svg
+      - type: file
+        path: com.geekbench.Geekbench6.appdata.xml
+      - type: file
+        path: com.geekbench.Geekbench6.desktop
+      - type: archive
+        url: https://cdn.geekbench.com/Geekbench-6.1.0-Linux.tar.gz
```

Comment:
Does the licence allow redistribution?

Reply:
I couldn't find a license. Wanted to figure out if it works first.

I guess I should move this to extra data then?

Reply:
Using extra-data now